Xamarin.Forms.Core/RuntimeNamePropertyAttribute.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Xamarin.Forms.Xaml
+{
+	[AttributeUsage(AttributeTargets.Class)]
+	public sealed class RuntimeNamePropertyAttribute : Attribute
+	{
+		public RuntimeNamePropertyAttribute(string name)
+		{
+			Name = name;
+		}
+
+		public string Name { get; private set; }
```

Comment:
turn on r#. it'll tell you you don't have to say `private set;`

Reply:
I have to turn if off when I'm working in your Xaml code, otherwise it formats everything correctly :)

Reply:
Fixed.